### PR TITLE
closes #84 source datasets doc@devel

### LINKED ISSUE #84: Documentation: Fix documentation of `source_datasets` argument in `derive_param_response()`
### Please select a category the issue is focused on?

Function Documentation

### Let us know where something needs a refresh or put your idea here!

Fix documentation of `source_datasets` argument in `derive_param_response()`. For this function there is no need to specify more than one dataset for `source_datasets`.

### PR DESCRIPTION
**Pull request by mattmarino**
Closes # .

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralonco/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@mattmarino reminder to include closes keyword at start of PR title and merge into devel rather than main. i updated these and will try take a look at this PR tomorrow.

File: R/derive_param_response.R
Comment (rossfarrugia) [inline R/derive_param_response.R:29]: Maybe re-word this section now as: 
```
It is implied that the response data and the data used in `pd_source` are in the same dataset `dataset`.
```

**Review by rossfarrugia — CHANGES_REQUESTED** [review]
thanks Matt! i left one comment.

i'd also like to add @bundfussr to the review when he's back next week as he made the issue so should have a better idea if this is as required. I couldn't figure out from the wording of the issue if we're saying we never expect 2 datasets here anymore (as you have updated) or maybe just that an extra note was needed saying you don't need to specify more than 1 (although for me the documentation as it is says that, so i think your updates are probably more on the mark).

File: R/derive_param_response.R
Comment (mattmarino) [reply R/derive_param_response.R]: Thanks @bundfussr, I've now updated to this. Do I need to create a new pull request for this change? I've pushed the change.

Comment (rossfarrugia) [reply R/derive_param_response.R]: no, its fine Matt as you pushed to same branch they all get added to this PR. there was one issue with the links auto-check below failing, but i just re-ran and it's fine now so must have been a network issue at the time. i'll approve now and once Stefan does we can merge this PR and move the issue to Done. thanks!

**Review by rossfarrugia — APPROVED** [review]

**Review by bundfussr — APPROVED** [review]

**Pull request merged**
